src/Components/AddToPlaylistModal/AddToPlaylistModal.jsx

```diff
@@ -0,0 +1,180 @@
+    const [ addToNewPlaylist, setAddToNewPlaylist ]           = useState(false)
+    const [ newPlaylistName, setNewPlaylistName ]             = useState("")
+
+    const addVideoToNewPlaylist = async () => {
+        const updatedUserInfo = await axios.post(
+            'https://videoztron.herokuapp.com/api/playlist/newplaylist',
+            {
+                video,
+                newPlaylistName
+            },
+            {
+                headers: {'x-access-token':localStorage.getItem('token')}
+            }
+        )
+
+        if(updatedUserInfo.data.status==="ok")
+        {
+            setAllPlaylists(updatedUserInfo.data.user.allPlaylists)
+            setNewPlaylistName("")
+            setShowPlaylistModal(false)
+            showToast("success","","Video successfully added to playlist!")
+        }
+    }
+
+    const addVideoToThisExistingPlaylist = async (event, playlistId) => {
+        const updatedUserInfo = await axios.patch(
+            "https://videoztron.herokuapp.com/api/playlist",
+            {
+                playlistId,
+                video
+            },
+            {
+                headers: {'x-access-token':localStorage.getItem('token')}
+            }
+        )
+
+        if(updatedUserInfo.data.status==="ok")
+        {
+            setAllPlaylists(updatedUserInfo.data.user.allPlaylists)
+            setShowPlaylistModal(false)
+            showToast("success","","Video successfully added to playlist!")
+        }
+    }
+
+    return (
+        <>
+        {
+            showPlaylistModal && (
+                <div 
+                    className="playlist-modal-container"
+                    onClick={(event)=>{
+                        event.preventDefault()
+                        event.stopPropagation()
+                    }}
+                >
+                    <div 
+                        className="playlist-modal"
+                        onClick={(event)=>{
+                            event.preventDefault()
+                            event.stopPropagation()
+                        }}
+                    >
+                        <div 
+                            className="playlist-modal-header-container"
+                            onClick={(event)=>{
+                                event.preventDefault()
+                                event.stopPropagation()
+                            }}
+                        >
+                            <h4>Save to . . .</h4>
+                            <GrClose 
+                                className="add-to-playlist-close-modal-btn" 
+                                style={{width:'20px', height:'20px'}}
+                                onClick={(event)=>{
+                                    event.preventDefault()
+                                    event.stopPropagation()
+                                    setShowPlaylistModal(prevState=> !prevState)
+                                    setAddToNewPlaylist(false)
```

Comment:
suggestion: You should create a handler function for onClick instead of writing here itself. This was suggested by Ankita in the code review.

Reply:
Yeah, it's just I felt 4 lines are too less to create a separate function for it.
And hence this makes code more trackable